### PR TITLE
hack: refactored code in update_owners.py

### PR DESCRIPTION
**Pull request by falcon78921**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind cleanup

**What this PR does / why we need it**:

Introduced ``pathlib`` as an alternative to ``os.path``. Updated the ``urllib2``
module to ``urllib.request``. Structured the code to align more with Python3.

If I'm missing anything, please let me know and I'll fix.

Signed-off-by: James McClune <jmcclune@mcclunetechnologies.net>

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

N/A

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by xichengliudui** [comment]
ok-to-test

**Comment by falcon78921** [comment]
Thanks @xichengliudui for allowing testing :)

**Comment by falcon78921** [comment]
@xichengliudui All of the tests passed. Does the PR look good for merging? If there's anything that needs fixed, please let me know.

**Comment by falcon78921** [comment]
@xichengliudui Should I just close this out?

**Comment by xichengliudui** [comment]
You don't have to close it

**Comment by xichengliudui** [comment]
/lgtm
cc fejta

**Comment by xichengliudui** [comment]
cc fejta

File: hack/update_owners.py [hunk 1]
```diff
@@ -14,22 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 import argparse
 import collections
 import csv
 import json
-import os
+from pathlib import Path
```
Comment (fejta) [inline hack/update_owners.py]: Please import modules not classes

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports

Comment (falcon78921) [reply hack/update_owners.py]: @fejta Ok, will do! :)

**Comment by falcon78921** [comment]
Thanks @xichengliudui :)

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

File: hack/update_owners.py [hunk 2]
```diff
@@ -14,22 +14,23 @@
...
 import json
-import os
+import pathlib
 import random
 import re
 import subprocess
 import sys
 import time
-import urllib2
+import urllib.request
```
Comment (spiffxp) [inline hack/update_owners.py]: doesn't exist in python 2

**Pull request merged**